mirage/serializers/application.js

```diff
@@ -1,3 +1,5 @@
 import { JSONAPISerializer } from 'ember-cli-mirage';
 
-export default JSONAPISerializer.extend();
+export default JSONAPISerializer.extend({
+  alwaysIncludeLinkageData: true,
```

Comment:
Can you comment what this is for?

Reply:
Will do so and push again!

Reply:
Ended up removing this; it was convenient for development but now it seems unnecessary.